src/main/java/com/example/travelservice/services/PersonalInfoServiceImpl.java

```diff
@@ -38,4 +43,22 @@ public Long findId(TravelerDto source) {
     public boolean personalInfoNotExists(PersonalInfo source) {
         return personalInfoRepository.findByEmailIs(source.getEmail()).isEmpty();
     }
+
+    @Override
+    public Traveler updateEmail(Long id, String email) {
+        Traveler traveler = travelerRepository.findById(id).orElseThrow(() ->
+                new NotFoundException("Traveler Not Found. For ID value: " + id.toString() ));
+        traveler.getPersonalInfo().setEmail(email);
+        return traveler;
+    }
+
+    @Override
+    public PersonalInfo updatePersonalInfo(Long id, PersonalInfo personalInfo) {
+
+        Traveler traveler = travelerRepository.findById(id).orElseThrow(() ->
+                new NotFoundException("Traveler Not Found. For ID value: " + id.toString() ));
+        Long travelerId = traveler.getPersonalInfo().getId();
+        personalInfo.setId(travelerId);
```

Comment:
Is it correct? Why you set travelerId as id of personal information?

Reply:
wrong naming, fixed